[NVSHAS-9680] Adding NeuVector/Rancher SSO Permission Resources Documentation

sunilarjun opened the pull request:
Adding documentation of mapped permission resources associated with NeuVector/Rancher SSO. Tied to NVSHAS-9680.

williamlin-suse commented on docs/02.deploying/03.rancher/03.rancher.md:
````diff
@@ -74,6 +74,142 @@ Note in the above screen shot, two Rancher users admin and gkosaka have been aut
+SystemConfig
+```
+
+Resource display for Rancher `Global/Cluster` Role Template pages:
````
Please remove lines 128~152
They are not available in Rancher's role creation -> resource drop-down menu

sunilarjun replied:
Thank you for reviewing, I've updated the content in this commit (https://github.com/neuvector/docs/pull/133/commits/423b3384a01d9b77f37dc3b502e47af701c8c3e7). The "Resources: NeuVector, Namespaced" is present for Rancher Global/Cluster Roles at lines 113-126.

williamlin-suse commented on docs/02.deploying/03.rancher/03.rancher.md:
````diff
@@ -74,6 +74,142 @@ Note in the above screen shot, two Rancher users admin and gkosaka have been aut
+SystemConfig
+```
+
+Resource display for Rancher `Project` Role Template pages:
````
Please remove lines 188~211
They are not available in Rancher's role creation -> resource drop-down menu

williamlin-suse commented on versioned_docs/version-5.4/02.deploying/03.rancher/03.rancher.md:
````diff
@@ -74,6 +74,142 @@ Note in the above screen shot, two Rancher users admin and gkosaka have been aut
+SystemConfig
+```
+
+Resource display for Rancher `Project` Role Template pages:
````
Please remove lines 188~211
They are not available in Rancher's role creation -> resource drop-down menu

williamlin-suse commented on versioned_docs/version-5.4/02.deploying/03.rancher/03.rancher.md:
````diff
@@ -74,6 +74,142 @@ Note in the above screen shot, two Rancher users admin and gkosaka have been aut
+SystemConfig
+```
+
+Resource display for Rancher `Global/Cluster` Role Template pages:
````
Please remove lines 128~152
They are not available in Rancher's role creation -> resource drop-down menu

williamlin-suse commented:
Who will update https://open-docs.neuvector.com/ about the changes in the PR?

sunilarjun commented:
> Who will update https://open-docs.neuvector.com/ about the changes in the PR?

Thank you for reviewing! I've addressed the feedback in this commit: https://github.com/neuvector/docs/pull/133/commits/423b3384a01d9b77f37dc3b502e47af701c8c3e7

For clarification this PR is handling the content update for the documentation at https://open-docs.neuvector.com/, and I will be handling the port of this PR to the product docs (https://documentation.suse.com/cloudnative/security/5.4/en/overview.html) as well once approved.
